.gitattributes

```diff
@@ -0,0 +1,2 @@
+ # Disable end of line conversions
+* -text
```

Comment:
It's because I was just _so sick_ of futzing about with line-endings I wanted to just get it working. As usual I prefer not to do bring extra requirements into tasks ... but i'm starting to wonder if i'm taking the best approach with that. 

My specific concern here is that to do anything more than change things so the current test expectations work will require creating more tests to confirm the new state, which seems out of scope.

Reply:
Do you think it's worth creating an issue for?

Reply:
Did what tom suggested here : https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/56/commits/80046b791b4d39794d046ea44cb2f4303bc0f76b
[plan]ned to do this everywhere here : https://github.com/OpenAssetIO/OpenAssetIO/issues/1200